app/models/exercise_participation.rb

```diff
@@ -0,0 +1,25 @@
+class ExerciseParticipation
+  include Neo4j::ActiveRel
+  include Neo4j::Timestamps
+
+  from_class :Person
+  to_class :Exercise
+  type :participated_in
```

Comment:
Relationship classes specify the kind of object on either end of the relationship.